app/lib/constants.js

```diff
@@ -1,39 +1,43 @@
-const crypto = require('crypto');
 
-const DB_NAME = process.env.DATABASE_NAME || 'dunedb';
 const APP_PORT = process.env.APP_PORT || 12313;
+const crypto = require('crypto');
+const DB_NAME = process.env.DATABASE_NAME || 'dunedb';
 
 /**
- * Here we set project-wide constants, pulling from the environment
- * and defaulting to safe values where applicable.
+ * Here we set project-wide constants, pulling from the environment and defaulting to safe values where applicable.
  * @constant
  */
-module.exports = {
+
+module.exports =
+{
   // Database Constants
   DB_NAME,
   DB_URL: process.env.DATABASE_URL || `mongodb://localhost:27017/${DB_NAME}`,
-  DB_CA: process.env.DATABASE_CA, // Certificate Authority and Intermediate certs for trust
-  DB_CERTKEY: process.env.DATABASE_CERTKEY, // Cert & Key in PEM format
-
+  DB_CA: process.env.DATABASE_CA,              // Certificate authority and intermediate certificates for trust
+  DB_CERTKEY: process.env.DATABASE_CERTKEY,    // Certificate & key in PEM format
+  
   // App Constants
   APP_PORT,
   NODE_ENV: process.env.NODE_ENV || 'development',
   BASE_URL: process.env.BASE_URL || `http://localhost:${APP_PORT}`,
   SESSION_SECRET: process.env.SESSION_SECRET || crypto.randomBytes(128).toString('hex'),
   M2M_SECRET: process.env.M2M_SECRET || crypto.randomBytes(256).toString('hex'),
-
+  
```

Comment:
You’ve introduced a lot of rogue whitespace in this file.  My assumption is that you’re not using VS Code yet?  I wrote this file to be compatible with future styling configuration so it really should not be changing at all unless you’ve found a bug or are adding a new feature.